blockbuilder/genesisState.go

```diff
@@ -0,0 +1,46 @@
+package blockbuilder
+
+import (
+	"encoding/json"
+	"errors"
+
+	"github.com/NethermindEth/juno/core"
+	"github.com/NethermindEth/juno/core/felt"
+)
+
+var (
+	ErrChainIDRequired = errors.New("ChainID is required")
+)
+
+type GenesisConfig struct {
+	ChainID                 string           `json:"chain_id"`
+	WhitelistedSequencerSet []felt.Felt      `json:"whitelisted_sequencer_set"`
+	State                   genesisStateData `json:"state"`
+}
+
+type genesisStateData struct {
+	Classes   map[string]genesisClassData    `json:"classes"`
+	Contracts map[string]genesisContractData `json:"contracts"`
+	Storage   map[string][]core.StorageDiff  `json:"storage"`
```

Comment:
I thought we reached consensus on having `FunctionCall`s?

Reply:
Yup, somehow slipped my mind :D

Reply:
Added them